Marlin/src/inc/Conditionals_post.h

```diff
@@ -723,6 +723,24 @@
         #define TEMP_0_SCK_PIN MAX31855_SCK_PIN
       #endif
 
+    #elif TEMP_SENSOR_0_IS_MAX31865
+      #if !PIN_EXISTS(TEMP_0_MISO) // DO
+        #if PIN_EXISTS(MAX31865_MISO)
+          #define TEMP_0_MISO_PIN MAX31865_MISO_PIN
+        #elif PIN_EXISTS(MAX31865_DO)
+          #define TEMP_0_MISO_PIN MAX31865_DO_PIN
+        #endif
+      #endif
+      #if !PIN_EXISTS(TEMP_0_SCK) && PIN_EXISTS(MAX31865_SCK)
+        #define TEMP_0_SCK_PIN MAX31865_SCK_PIN
+      #endif
+      #if !PIN_EXISTS(TEMP_0_MOSI) && PIN_EXISTS(MAX31865_MOSI) // MOSI for '65 only
+        #define TEMP_0_MOSI_PIN MAX31865_MOSI_PIN
+      #endif
+
+    /*
+    #not sure if this should be referencing 1, makes sense for it to be 0.  
```

Comment:
Is that because these are expected to share an SPI channel? It doesn't have to be a shared SPI, if that is the concern.

Reply:
The reference for this set of conditionals is #22196.